test/JDBC/expected/BABEL-5119-vu-verify.out

```diff
@@ -743,3 +743,22 @@ GO
 ~~ROW COUNT: 1~~
 
 -- terminate-tsql-conn user=login_babel5119_2 password=12345678
+
+-- tsql database=BABEL5119_db
+CREATE SCHEMA S1;
```

Comment:
Make sure to drop the created objects.

Reply:
I am dropping the entire database in cleanup